include/SystSkimmer.h

```diff
@@ -137,7 +138,7 @@ struct TupleMaker {
   std::mutex mutex;
   std::condition_variable cond_var;
 };
-
+/*
```

Comment:
Remove the commented part which is now defined in `Utilities.h`.

Reply:
ok